.github/workflows/bindists.yaml

```diff
@@ -47,14 +50,14 @@ jobs:
       image: ${{ matrix.image }}
     steps:
       - name: Install requirements
+        shell: sh
         run: |
           ${{ matrix.installCmd }} curl bash git ${{ matrix.toolRequirements }}
       - uses: actions/checkout@v3
       - name: Install ghcup
         run: |
           curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
           echo ~/.ghcup/bin >> $GITHUB_PATH
-          ~/.ghcup/bin/ghcup --version
```

Comment:
Here, we run in base docker containers, which don't contain ghcup, but further down (for macos and windows), I agree that it is nice to have that guarantee. I only removed it because `~/.ghcup/bin/ghcup` isn't actually the correct path on windows, but it should be easily possible to work around that :+1:

Reply:
We want something like `which ghcup`